src/common/post.ts

```diff
@@ -145,3 +147,58 @@ export const createSquadWelcomePost = async (
     origin: PostOrigin.UserGenerated,
   });
 };
+
+export type EditablePost = Pick<
+  FreeformPost,
+  'title' | 'content' | 'image' | 'contentHtml'
+>;
+
+export type CreatePost = Pick<
+  FreeformPost,
+  'title' | 'content' | 'image' | 'contentHtml' | 'authorId' | 'sourceId' | 'id'
+>;
+
+export const createFreeFormPost = async (
```

Comment:
Another thought, I see that this function doesn't really create the freeform as it just saves it to the db. It doesn't handle the markdown rendering, mentions, and so on. Isn't it confusing?

Reply:
Good point. Let's use the most accurate label. Will refactor 🚢